CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+= Contributing to the Pacemaker project =
+
+If you find Pacemaker useful and wish to support the project, you can:
+
+* Spread the word (on blogs, social media, mailing lists, Q&A websites, etc.).
+* Join and participate in the [mailing lists](http://clusterlabs.org/mailman/listinfo/).
+* Report [bugs and new feature requests](http://bugs.clusterlabs.org/).
+* Contribute documentation, bug fixes, or features to the code base.
+
+If you would like to contribute code base changes, please read
+[Pacemaker Development](http://clusterlabs.org/doc/en-US/Pacemaker/1.1-pcs/html-single/Pacemaker_Development/index.html)
+for detailed information about pull requests and policies.
```

Comment:
The goal is to have the top-level documentation be useful.

NEWS and TODO went away because they weren't useful at all. AUTHORS was merged into README because a full list of authors isn't practical anymore.

INSTALL was separated from README because people who compile from source commonly look for separate install instructions there, and because it was making the README much larger than necessary.

CONTRIBUTING was separated so github could pick it up. It was left in the top level because it is of wider interest than just gitub links. A separate CONTRIBUTING for github only was avoided because it is would be unnecessary duplication to maintain.